src/Parsers/LatLongParserBase.php

```diff
@@ -33,22 +33,25 @@ abstract class LatLongParserBase implements ValueParser {
 	 */
 	public const OPT_SEPARATOR_SYMBOL = 'separator';
 
-	/**
-	 * @var ParserOptions
-	 */
-	private $options;
+	private ParserOptions $options;
 
 	public function __construct( ?ParserOptions $options = null ) {
 		$this->options = $options ?: new ParserOptions();
 
-		$this->options->defaultOption( ValueParser::OPT_LANG, 'en' );
+		$this->defaultOption( ValueParser::OPT_LANG, 'en' );
```

Comment:
What's the reason for preferring the private `defaultOption` method here to repeated calls to `withDefaultOption`? Is it a wish to avoid repeated re-allocation of the options object?

Reply:
It was the quickest fix